src/client/datascience/notebook/notebookEditorProvider.ts

```diff
@@ -76,6 +76,10 @@ export class NotebookEditorProvider implements INotebookEditorProvider {
         @inject(IExtensions) private readonly extensions: IExtensions
     ) {
         disposables.push(this);
+        // Register notebook documents that are already open e.g. after a workspace reload
+        for (const notebookDocument of this.vscodeNotebook.notebookDocuments) {
+            void this.onDidOpenNotebookDocument(notebookDocument);
+        }
```

Comment:
This is the actual fix--notebookEditorProvider wasn't aware of notebookDocuments that were already loaded.

Reply:
So we've always had this issue! Whoa

Reply:
Actually, I debugged the release branch in VS Code 1.59.1, and it looks like VS Code has stopped firing `onDidOpenNotebookDocument` for already-open NotebookDocuments in Insiders, so previously this code wasn't even necessary. i.e. this was a behavior change in VS Code, not the Jupyter extension.